MqttClient_Hello/app/application.cpp

```diff
@@ -11,12 +11,25 @@
 void startMqttClient();
 void onMessageReceived(String topic, String message);
 
-Timer procTimer;
+Timer procTimer, mqttTimer;
 
 // MQTT client
 // For quickly check you can use: http://www.hivemq.com/demos/websocket-client/ (Connection= test.mosquitto.org:8080)
 MqttClient mqtt("test.mosquitto.org", 1883, onMessageReceived);
 
+// Check for MQTT Disconnection
+void IRAM_ATTR checkMQTTDisconnect(TcpClient& client, bool flag){
+	
+	// Called whenever MQTT connection is failed.
+	if (flag == true)
+		Serial.println("MQTT Broker Disconnected!!");
+	else
+		Serial.println("MQTT Broker Unreachable!!");
+	
+	// Restart connection attempt after few seconds
+	mqttTimer.initializeMs(2 * 1000, startMqttClient).start(); // every 2 seconds
```

Comment:
will this run every 2 seconds forever? should the timer not be stopped once it is connected?